src/Import/src/FMI3/fmi3_import_capi.c

```diff
@@ -195,6 +195,13 @@ jm_status_enu_t fmi3_import_instantiate_model_exchange(fmi3_import_t* fmu,
     fmi3_instance_t c;
     if (!resourcePath)
         resourcePath = fmu->resourcePath;
+    
+    fmu->instanceName = fmu->callbacks->malloc(strlen(instanceName)+1);
+    if (!fmu->instanceName) {
+        jm_log_fatal(fmu->callbacks, module, "Could not allocate memory");
+        return fmi3_status_fatal;
+    }
+    strcpy(fmu->instanceName, instanceName);
```

Comment:
Fixes the TODO in fmi3_log_forwarding (the problem is that FMI2 logger callback has `const char* instanceName` arg, but that's gone in FMI3, so we need to save it at instantiation instead to keep old behavior).